File: Problems/NepsAcademy/e86.py

def resp(a, b):
    
    if a + b >= 14:
        return "Aprovado"

    if a + b < 8:
        return "Reprovado"
    
    return "Recuperacao"





A, B = map(float, input().split())

print(resp(A, B))

File: Estudos/EstruturasCondicionais/cpp/e86.cpp

#include <bits/stdc++.h>

using namespace std;

int main(){
    ios::sync_with_stdio(false);
    cin.tie(nullptr);

    double a, b;
    string resp = "Recuperacao";

    cin >> a >> b;

    if ((a + b) >= 14.0){
        resp = "Aprovado";
    } else if((a + b) < 8){
        resp = "Reprovado";
    }

    cout << resp << '\n';

}